startly_style.css

```diff
@@ -0,0 +1,218 @@
+@import url('http://fonts.googleapis.com/css?family=Economica');
```

Comment:
I didn't know that we could do it like this - cool! I've been putting it in the html, but this seems like a nicer way.